projects/schematics/src/shared/lib-configs/integration-libs/segment-refs-schematics-config.ts

```diff
@@ -26,19 +26,22 @@ export const SEGMENT_REFS_SCHEMATICS_CONFIG: SchematicConfig = {
   folderName: SEGMENT_REFS_FOLDER_NAME,
   moduleName: SEGMENT_REFS_MODULE_NAME,
   featureModule: {
-    name: SEGMENT_REFS_MODULE,
+    name: SEGMENT_REFS_ROOT_MODULE,
     importPath: SPARTACUS_SEGMENT_REFS,
   },
   rootModule: {
-    importPath: SPARTACUS_SEGMENT_REFS, //as SPARTACUS_SEGMENT_REFS_ROOT is same as SPARTACUS_SEGMENT_REFS, no extra module/features are present
-    name: SEGMENT_REFS_MODULE,
-    content: `${SEGMENT_REFS_MODULE}`,
+    importPath: SPARTACUS_SEGMENT_REFS_ROOT,
+    name: SEGMENT_REFS_ROOT_MODULE,
+    content: `${SEGMENT_REFS_ROOT_MODULE}`,
   },
+  /* Through Spartacus Segment-refs code doesnot have a dependency on Personalization,
+  backend occ api of Segment-refs requires Personalization to be enabled , hence adding this dependency
+  If Personalization library is not installed, Personalization & Segment-refs won't apply */
   dependencyFeatures: [TRACKING_PERSONALIZATION_FEATURE_NAME],
   importAfter: [
     {
       markerModuleName: PERSONALIZATION_MODULE,
-      featureModuleName: SEGMENT_REFS_MODULE,
+      featureModuleName: SEGMENT_REFS_ROOT_MODULE,
     },
   ],
 };
```

Comment:
Do we _really_ need this config `importAfter` (which creates lazy loadable wrapper modules)?
I guess we don't want to create a lazy-loadable wrapper module for personalization feature and we don't want to  append SegmentRefsRootModule after the original PersonalizationModule. What do you think?

Reply:
done